dali/daliadmin/daadmin.cpp

```diff
@@ -273,12 +273,7 @@ void set(const char *path,const char *val)
 
 //=============================================================================
 
-void get(const char *path)
-{
-    getValue(path, nullptr);
-}
-
-void getValue(const char *path, StringBuffer *value)
+void get(const char *path, StringBuffer *value)
```

Comment:
related to 2nd commit comments.
It would be better to unify on a single getValue that always filled a StringBuffer,
then what is done with the result (log vs return to response) is something handled by daliadmin, vs esp service.

Reply:
Got the idea. The implementation details may be different for each esp service methods. May change the get() in the PR of implementing the esp service get().